treeherder/perf/models.py

```diff
@@ -47,6 +47,8 @@ class PerformanceSignature(models.Model):
     parent_signature = models.ForeignKey('self', on_delete=models.CASCADE, related_name='subtests',
                                          null=True, blank=True)
     has_subtests = models.BooleanField()
+    suite_public_name = models.CharField(max_length=50, blank=True)
```

Comment:
Now that we have the chance to restrict these names, we should make proper use of it.
Let's set `max_length=30`.

Reply:
`blank=True` means we have some interest in what we get from frontend. But we don't care about this deserialization direction. We're only interested in displaying aspects here.
Thus, I suggest we replace it with `null=True`.

Reply:
A short comment would really help here. Something like "we treat suite & test as human unreadable identifiers; these are their human readable form".